### PR TITLE
Jkpractice

### PR DESCRIPTION
**Pull request by msjkraft**
This is my attempt at forking, fixing, and pulling for Skillcrush 105.

### CODE REVIEW

**Comment by redrambles** [comment]
Hey @msjkraft !

This is looking good! The only thing we need is that you link to your css file (`../css/jkraft-advice.css`) from your advice file. Right now it is only linking to (`../css/style.css`).

Once you've made that change, please add, commit and then push up your branch once more, which will automatically update this pull request. Then we should be all set to merge in your code! :0

Let us know if you have any questions!

Cheers,
Ann

**Comment by msjkraft** [comment]
Ok. I linked the css file and I hope that I added, committed, and pushed the branch.

Thanks,
Jen

File: advice/jkraft-advice.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" type="text/css" href="../css/style.css">
```
Comment (redrambles) [inline advice/jkraft-advice.html]: This is the link that needs to be changed. :)

Comment (redrambles) [reply advice/jkraft-advice.html]: Hey @msjkraft! You mention earlier in this conversation that you had changed the link to your CSS file - but that edit never made it to your pull request. Can you double-check the result of `git status` to make sure that you don't have any changes that still needs to be added and/or committed? Once the result of git status is clean, please try pushing up your branch again with `git push origin jkpractice`. 

Keep us posted and let us know if you have more questions!

Cheers,
Ann

**Comment by acodeinprogress** [comment]
Hey @msjkraft,

We haven't heard from you in a while so I'm going to close this pull request. Feel free to submit a new one when you're ready!

xoxo,
Sharon

**Comment by msjkraft** [comment]
Sorry. I was on vacation and didn't take my computer for the first time.
Ann has been helping me in Slack and I think I finally got it. Can you
please check when you get a chance?

Thanks,
Jen

**Pull request closed without merging**